src/clear-clipboard.js

```diff
@@ -0,0 +1,20 @@
+const clipboardy = require('clipboardy')
+
+module.exports = {
+  clearClipboardHistory() {
```

Comment:
Since this module is exposing this `clearClipboardHistory` method, does it make sense for the file to be called `clear-clipboard.js`? Should it just be `clipboard.js`?

Alternatively, this module could expose the function as its only export. Then the whole module would just be the one function and the name would make more sense. But overall, I like renaming the file better.

Reply:
That is a good point, I also like renaming the file better